File: Basics of Computing and Programming/bmiimperial.py
def main():
    weight = float(input("Please enter weight in pounds: "))
    height = float(input("Please enter height in inches: "))
    print(f"BMI is: {(weight * 0.453592) / (height * 0.0254) ** 2}")

if __name__ == "__main__":
    main()

File: Introduction to Programming in C++/Week3/bmiimperial.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
    float height;
    float weight;

    // Input height and weight
    cout << "Please enter weight in pounds: ";
    cin >> weight;
    cout << "Please enter height in inches: ";
    cin >> height;
    
    // Calculate BMI
    weight *= 0.453592;
    height *= 0.0254;
    float bmi = weight / pow(height, 2);
    cout.flags(ios::fixed);
    cout.precision(2);
    cout << "BMI is: " << bmi << endl;
    return 0;
}